Styles.css

```diff
@@ -0,0 +1,92 @@
+* {
+  margin: 0%;
+  padding: 0%;
+}
+
+body {
+  margin-right: 486.5px;
+  margin-left: 486.5px;
+}
+
+.header_section {
+  background-color: #3c3a39;
+  height: 92px;
+  position: relative;
+  display: flex;
+}
+
+#unicon {
+  position: absolute;
+  inset: 63px 19px 16px 323px;
+}
+
+.headline_section {
+  background-color: #1c1a19;
+  background-image: url(Sources/Group.png);
+  background-repeat: no-repeat;
+  background-position: 244px 322px;
+  height: 590px;
+  position: relative;
+  display: flex;
+}
+
+#name {
+  color: white;
+  position: absolute;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 19px;
+  text-decoration: none;
+  text-align: center;
+  inset: 56px 240px 12px 24px;
+}
+
+#bkg_illustrations {
+  position: absolute;
+  inset: 316px 0 0 243px;
+}
+
+.social_media {
+  position: absolute;
+  inset: 430px 0 0 24px;
+  list-style: none;
+}
+
+.social_media li {
+  display: inline;
+}
+
+#primary_text {
+  color: #ff6b00;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  position: absolute;
+  inset: 120px 0 0 24px;
+}
+
+#supporting_text {
+  color: #fff5e1;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  position: absolute;
+  inset: 256px 0 0 24px;
+}
+
+#description {
+  color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  position: absolute;
+  inset: 295px 0 0 24px;
```

Comment:
- [ ] Kindly make sure you use the correct `color` and `typo` as provided in the Figma design. As you can see how these colors you have used mess up your project.
![Screenshot from 2022-09-23 09-06-02](https://user-images.githubusercontent.com/97100091/191908481-3199ae61-fedf-423a-bf6a-4715226489a8.png)